src/calculateRentalCost.js

```diff
@@ -28,7 +28,23 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const PRICE_PER_DAY = 40;
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCOUNT = 50;
+  const SHORT_TERM = 3;
+  const SHORT_TERM_DISCOUNT = 20;
```

Comment:
It's better to save constants outside the function to improve code readability and maintainability, enhance reusability, prevent repetition of constant declarations, provide flexibility for modifications, and facilitate future updates or adjustments to pricing rules or discounts.